src/UI/Components/ServiceInstanceForm/Components/FieldInput.tsx

```diff
@@ -40,6 +39,12 @@ export const FieldInput: React.FC<Props> = ({
   getUpdate,
   path,
 }) => {
+  const getEnumUpdate = useCallback(
```

Comment:
Also, maybe make sure to add some docstrings to explain why you had to use memoization :D